webviz_examples/content/how_was_this_made.md

```diff
@@ -9,3 +9,17 @@ It might look long, but the idea is very simply, and as abstract as it can get:
 - **Specify the content** (i.e. which dashboard you want to appear) on each page.
 - Each dashboard takes some (optional) arguments that you can provide if you want 
   to **override default settings**.
+
+Webviz will create the application for you, from the configuration file, when
+you either run
+```bash
+webviz build your_config.yml
+```
+or
+```bash
+webviz build your_config.yml --portable ./some_output_folder
+python ./some_output_folder/webviz_app.py
+```
+The first one will create a "live app" which reloads and updates when you change
+the configuration file, the latter will create a portable app where any data aggregation
+is done as a preprocessing step during the build process.
```

Comment:
Specify clearer that the portable option often is recommended for larger apps due to performance and making sure that required data isn't lost? Guess some could be tempted by the "live" features like auto-reload, but get annoyed by the time it takes. Just to avoid any possible misunderstanding.

Reply:
👍 Something like in the newest commit?